.github/workflows/test-build.yml

```diff
@@ -173,6 +173,14 @@ jobs:
             -f user-office-gateway/docker-compose.e2e.yml \
             -f "$REPO_DIR_NAME/docker-compose.e2e.yml" \
             up --exit-code-from cypress
+
```

Comment:
How can we access the screenshots afterwards?

Reply:
If you click on the action name above the action steps (`Build, Test and Push` in this case) you will be redirected to the selected Github action run page, where is an `Artifacts` panel, under that should be a `cypress-screenshots` file.
You can access from PR Actions tab or by selecting a run under the repo's Action tab.

Reply:
I included an example link: https://github.com/UserOfficeProject/user-office-scheduler-frontend/actions/runs/385111575

Reply:
Nice